.nuspec/Microsoft.Maui.Controls.SingleProject.targets

```
@@ -68,6 +68,44 @@
     <TizenSharedPrefix>$(TizenProjectFolder)shared</TizenSharedPrefix>
   </PropertyGroup>
 
+  <!-- Add metadata indicating that the platform-specific files are not part of every build configuration. -->
+  <ItemGroup Condition=" '$(SingleProject)' == 'true' ">
+    <Compile Update="$(PlatformsProjectFolder)**/*$(DefaultLanguageSourceExtension)">
+      <ExcludeFromCurrentConfiguration>true</ExcludeFromCurrentConfiguration>
+    </Compile>
+  </ItemGroup>
+
+  <!-- Add metadata for the files that are actually part of the current build configuration. -->
+  <ItemGroup Condition=" '$(SingleProject)' == 'true' and '$([MSBuild]::GetTargetPlatformIdentifier($(TargetFramework)))' == 'android' ">
```

Since this is an item group, it happens in a later MSBuild pass than properties. I think you could do this instead and it would work:
```suggestion
  <ItemGroup Condition=" '$(SingleProject)' == 'true' and '$(TargetPlatformIdentifier)' == 'android' ">
```

---

Done.